kitchen.dokken.yml

```diff
@@ -224,10 +232,22 @@ suites:
       splunk:
         disabled: true
 
+  - name: upgrade_server
+    run_list:
+      - recipe[test::default]
+      - recipe[chef-splunk]
+      - recipe[chef-splunk::upgrade]
+    attributes:
+      splunk:
+        upgrade_enabled: true
+        accept_license: true
+        is_server: true
+
   - name: server_lwrps
```

Comment:
any chance while you're in here you can rename this to 'server_resources'. We're wiping LWRPs from the Chef vocab.